homeassistant/components/fyta/quality_scale.yaml

```diff
@@ -0,0 +1,90 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: No custom action.
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions:
+    status: exempt
+    comment: No custom action.
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: No custom action.
+  reauthentication-flow: done
+  parallel-updates:
+    status: exempt
+    comment: |
+      Coordinator and only sensor platform.
+
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: |
+      No options flow.
+
+  # Gold
+  entity-translations: done
+  entity-device-class: done
+  devices: done
+  entity-category: done
+  entity-disabled-by-default:
+    status: exempt
+    comment: No noisy entities.
+  discovery:
+    status: exempt
+    comment: Fyta device cannot be discovered.
```

Comment:
The devices connect to a wifi hub right? Did you check if the manufacturer has an OUI for the hub's mac address?

Reply:
I haven't thought of this, because the integration does not connect directly to the hub. But you could be right, if there is a hub in the network, it might get discovered. I will investigate on this.

Reply:
According to information from FYTA, the hub has no OUI. Do you see an alternative for discovery? If not I would set the conversation to resolved.

Reply:
As discussed, I added a PR for dhcp discovery: https://github.com/home-assistant/core/pull/132185